.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: 🐛 Bug Report
+description: Report a reproducible bug.
+title: "Bug report: <short description>"
+labels: ["bug", "unconfirmed"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting the issue you are facing. Please complete the following so we can have the details necessary to assist you.
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: Prerequisites
+      description: Please confirm before submitting a new issue
+      options:
+        - label: I am running the [latest version of the Browser Extension Wallet](https://github.com/fetchai/fetch-wallet/releases).
+          required: true
+        - label: I checked the [documentation](https://docs.fetch.ai/browser-extension-wallet/) and found no answer to my problem.
```

Comment:
https://docs.fetch.ai/browser-extension-wallet/ : 404 - Not found

Reply:
for now we can use this one:
https://docs.fetch.ai/getting-started/how-to-use-browser-wallet/

Reply:
@VijaySharma815 I applied your suggestion but that also currently looks like a 404 to me.

Reply:
@bryanchriswhite we just merged some changes on docs so the updated URL should be:
https://docs.fetch.ai/basics/wallet/getting_started/